app/controllers/tag_controller.rb

```diff
@@ -11,6 +11,12 @@ def index
 
     @title = I18n.t('tag_controller.tags')
     @paginated = true
+
+    order_string = "count DESC"
```

Comment:
I would put this into a private function within the controller.  e.g. 
```
def order_string
  params[:order] === "asc" ? "count ASC" : "count DESC"
```

I would also perhaps abstract `count ASC` and `count DESC` into constants themselves.  e.g. `COUNT_ASC = "count ASC"` and `COUNT_DESC = "count DESC"`.  Or put them also into private functions, e.g. 

```
def count_asc
  "count ASC"
end 
```

This way, if we ever need to change `count DESC` to some other string, we'll have it all in one place, either in the constant or in the function, and we won't have to chase it around everywhere. :)